lib/kafo/configuration.rb

```diff
@@ -240,6 +240,23 @@ def answers
       @data
     end
 
+    def run_migrations
+      migrations = Kafo::Migrations.new(migrations_dir)
+      migrations.load_migrations
```

Comment:
You mean  the run_migrations is invoked in initializer? That would make good sense.